feat(): previews custom template

locinus opened the pull request:
And here we are !
This is the documentation related to the previews customization.
I hope everything is clear, I'm somewhat sleepy today!

locinus committed:
feat(): previews custom template

Ks89 commented:
Thank you.
It's ok, the only thing that I suggest to change is the section `Add additional data in the ModalImage objects` because it can be another live example. 
I suggest to create another dedicated page for the live example with preview template + additional data.

locinus committed:
feat(): previews custom template + additional data
Splitted the initial example page into 2 pages:
- previews template
- previews template + additional data

locinus commented:
So be it.

Ks89 commented:
Thank you. Merged
